app/src/main/java/com/example/vanahel/tasksmanagerapplication/tab/AllTabFragment.java

```diff
@@ -0,0 +1,71 @@
+package com.example.vanahel.tasksmanagerapplication.tab;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import com.example.vanahel.tasksmanagerapplication.R;
+import com.example.vanahel.tasksmanagerapplication.adapter.TasksArrayAdapter;
+import com.example.vanahel.tasksmanagerapplication.contracts.TabFragmentContract;
+import com.example.vanahel.tasksmanagerapplication.dao.DAOManager;
+import com.example.vanahel.tasksmanagerapplication.dao.TaskDAO;
+import com.example.vanahel.tasksmanagerapplication.loader.callback.AsyncTaskLoaderCallbacks;
+import com.example.vanahel.tasksmanagerapplication.presenter.TabFragmentPresenter;
+
+public class AllTabFragment extends Fragment implements TabFragmentContract.View{
+
+    private AsyncTaskLoaderCallbacks asyncTaskLoaderCallbacks;
+    private TasksArrayAdapter allTasksArrayAdapter;
+    private static AllTabFragment allTabFragment;
```

Comment:
Don't use static links on Fragments, Activities and other Android stuff that might have UI impact and\or have link on Context. Because it might lead to Memory Leaks.